src/project-builder.ts

```diff
@@ -29,33 +28,12 @@ export const projectBuilder = async function () {
     return;
   }
 
-  // @ts-ignore https://github.com/enquirer/enquirer/issues/379
-  const confirmer = new prompt.Confirm({
-    name: "mkdirartifacts",
-    message: "Do you want to clone Tableland's Validator repository?",
-  });
-
-  const mkdirArtifacts = await confirmer.run();
-
-  if (!mkdirArtifacts) {
+  // check for existing config and bail if it exists
+  if (configExists()) {
     console.log(
-      chalk.yellow(`${chalk.bold("Not")} cloning the Validator repo.`)
-    );
-  } else {
-    console.log(
-      chalk.yellow(`Creating a ${chalk.bold("tableland-artifacts")} directory.`)
-    );
-    // make an artifacts directory
-    spawnSync("mkdir", ["tableland-artifacts"]);
```

Comment:
Now that the local network can run without cloning anything via git I don't think it makes sense to do this in the cli anymore.  If someone wants to specify the location of the go-tableland and evm-tableland repos they should have no problem doing the cloning themself.